visualization_and_plotting/readme.md

```diff
@@ -0,0 +1,43 @@
+#Simple horizontal Bar Plots
+---
+#R
+```r
+library(dplyr)
+library(magrittr)
+library(ggplot2)
+
+iris %>%
+    dplyr::group_by(Species) %>%
+    dplyr::summarise(Sepal.Length = mean(Sepal.Length)) %>%
+    ggplot(aes(Species, Sepal.Length)) + geom_bar(stat = 'identity') + 
```

Comment:
Avant style guide suggests to always use package::function nomenclature.  So ggplot2::ggplot here all the way through.  Then you don't need to issue the library commands.